ECL global keyring section (#107)
hpcng/singularity-admindocs: configfiles.rst

    To temporarily permit the use of legacy insecure signatures, set
    ``legacyinsecure = true`` in ``ecl.toml``.

> Manage ECL public keys

Reviewer: ```suggestion
Managing ECL public keys
```

> ======================

Reviewer: ```suggestion
=======================
```

This is what's causing CI to fail.

Author: fixed by https://github.com/hpcng/singularity-admindocs/pull/107/commits/6c8aafa586e56bc2a35dc65c5b05ab453ebf549c

> Since Singularity 3.6.0, public keys associated to ECL key fingerprints

Reviewer: ```suggestion
In Singularity 3.6, public keys associated with fingerprints specified in ECL rules
```

> are required to be present in user's local keyring which is not very

Reviewer: ```suggestion
were required to be present in user's local keyring which is not very
```

convenient. Singularity 3.7.0 provides a mechanism to administrators for managing
a global keyring that ECL uses during signature verification, for that purpose a
``--global`` option was added for:

  * ``singularity key import`` (root user only)
  * ``singularity key pull`` (root user only)
  * ``singularity key remove`` (root user only)
  * ``singularity key export``
  * ``singularity key list``

.. note::

>     For obvious security reasons, it is not possible to import private keys

Reviewer: It won't be obvious to some people who are not really aware of the nature of asymmetric keys. Would be better to remove "obvious" or give a specific reason.

Author: fixed
